### DISCUSSION TITLE
Support for leaveDotGit option | Nixpacks builds

### DISCUSSION THREAD

**Discussion by EternalC0der** [coollabsio/coolify#4570] [Feature Requests]
Hello,

I was trying to include the .git directory in the build-time so I could read some meta-data about the last commit and use it in the version section in our frontend. (we do a little console.log to see the last commit hash and date)

But since the .git directory is not present (I assume this because I had errors about missing .git directory)
I could not fetch any versioning details from the .git directory.

I would really appreciate if `leaveDotGit` from Nixpacks was an option to enable it or disable it,
So then I could utilize the .git folder in build time.

Thanks.

**Comment by jesspinklet** [comment]
I would like to see this as well 👀
